### ISSUE TITLE
Ability to resume program after it was being stopped

### ISSUE THREAD

**Issue by gshep** [gear-tech/gear#2605]
### Problem to Solve

after #2301 programs will be stopped after the specified amount of blocks. #2509 adds ability to prolong programs' rent. The last one thing is to provide feature to users to resume programs paused earlier.

### Possible Solution

-

**Comment by gshep** [comment]
duplicate of #2564
